From 3daf9fc66400b566f60626ac8aa1c0f0a8f3d35a Mon Sep 17 00:00:00 2001
From: Andreas Schultz <andreas.schultz@travelping.com>
Date: Mon, 20 Apr 2020 09:45:24 +0200
Subject: [PATCH 2/7] Only install hyperscan on amd64

Hyperscan is only supported on Intel. Keep it simple and
test only for x86_64 (who is still using 32bit anyway?)

Change-Id: I73707d33860820e4cee0b95575c11756236466a4
---
 Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index a2eab953a..841eb3bdf 100644
--- a/Makefile
+++ b/Makefile
@@ -72,7 +72,9 @@ DEB_DEPENDS += cmake ninja-build uuid-dev python3-jsonschema python3-yaml
 DEB_DEPENDS += cmake ninja-build uuid-dev python3-jsonschema python3-yaml yamllint
 DEB_DEPENDS += python3-venv  # ensurepip
 DEB_DEPENDS += python3-dev   # needed for python3 -m pip install psutil
+ifeq ($(MACHINE),x86_64)
 DEB_DEPENDS += libhyperscan-dev
+endif
 # python3.6 on 16.04 requires python36-dev
 
 LIBFFI=libffi6 # works on all but 20.04
@@ -144,7 +146,9 @@ endif
 
 # +ganglia-devel if building the ganglia plugin
 
+ifeq ($(MACHINE),x86_64)
 RPM_DEPENDS += hyperscan-devel
+endif
 RPM_DEPENDS += chrpath libffi-devel rpm-build
 
 RPM_DEPENDS_DEBUG  = glibc-debuginfo e2fsprogs-debuginfo
@@ -159,7 +163,9 @@ RPM_SUSE_BUILDTOOLS_DEPS += clang cmake indent libtool make ninja python3-ply
 
 RPM_SUSE_DEVEL_DEPS = glibc-devel-static libnuma-devel
 RPM_SUSE_DEVEL_DEPS += libopenssl-devel openssl-devel mbedtls-devel libuuid-devel
+ifeq ($(MACHINE),x86_64)
 RPM_SUSE_DEVEL_DEPS += hyperscan-devel
+endif
 
 RPM_SUSE_PYTHON_DEPS = python-devel python3-devel python-pip python3-pip
 RPM_SUSE_PYTHON_DEPS += python-rpm-macros python3-rpm-macros
-- 
2.28.0

